src/manager/node.rs

```diff
@@ -254,6 +264,8 @@ where
     }
 
     fn handle_node_event(&mut self, event: NodeManagerEvent) -> ArconResult<()> {
+        gauge!("nodes", self.nodes.len() as f64 ,"node_manager" => self.state_id.clone());
```

Comment:
Probably don't want to update the gauge every ``handle_node_event`` call.  As the nodes are currently static, calling the gauge!(..) at on_start would be suitable. However, I think the manager currently starts before actually adding the nodes into ``self.nodes``.

For it to work at ``on_start``, the following change has to happen:

```rust
// Start NodeManager
  app.ctrl_system()  // Line 270 src/dataflow/constructor.rs
     .start_notify(&manager_comp)
     .wait_timeout(std::time::Duration::from_millis(2000))
     .expect("Failed to start NodeManager");
// has to be moved down to after line 320.
```